### DISCUSSION TITLE
add woocommerce plugin with atomic site

### DISCUSSION THREAD

**Discussion by steve-lgtm** [Automattic/wp-calypso#69111] [Q&A] [answered]
How can I start with atomic and Woocommerce plugin installed? I can't add plugin because they asked me to upgrade my plan. I'm totally new in this project. Can you help me please ?

**Comment by lynnbgriffith** [comment]
Hello! You’ll probably want to reach out to [WordPress.com support](https://support.wordpress.com). This GitHub repository is where we develop the open source Calypso UI used on WordPress.com, but we’re not really equipped to help out with your specific WP.com plan or plugins. There are a couple good support articles that might help you get started with Woo on WP.com:
- https://en.support.wordpress.com/ecommerce/
- https://wordpress.com/support/new-woocommerce-experience-on-wordpress-dot-com/

**Reply by steve-lgtm** [reply]
Okay I just try different question. How can I contribute when I can't add new plugins because wordpress say I need to upgrade to the PRO plan?
![Screenshot 2022-11-05 142702](https://user-images.githubusercontent.com/60081152/200122260-c7bbbb64-af67-4431-8a8a-7b2c0e46257a.png)

**Comment by lynnbgriffith** [comment, marked as answer]
If you're hoping to contribute to Calypso, this repo's README has some information about how to run it locally and get started with development. Note that Calypso isn't actually a WordPress plugin, but a JavaScript-based client that uses the REST API to communicate with WordPress.com and Jetpack sites.
If you're hoping to contribute to WooCommerce, you'd likely start with their contributor guidelines:
https://github.com/Automattic/woocommerce/blob/security/DEVELOPMENT.md
I'd personally also try setting up a local environment for that development work myself. However, if you wanted to use WordPress.com for WooCommerce development, you'd need a Business plan in order to use custom plugins and themes. Those plans provide an isolated environment safely allowing use of custom code. They also provide SSH access, which can be really handy for many development workflows.
Best of luck!

**Comment by johanjonikhan** [comment]
Thank you for sharing your question! Your post is very logical and informative, especially for someone new to this project. To get started with Atomic and WooCommerce, you may need to check if your current hosting plan supports plugin installations. Many hosting providers restrict plugin installations on basic plans, so upgrading to a plan that allows full customization is usually necessary. Additionally, ensure you’ve installed the WooCommerce plugin correctly after upgrading.

For those exploring opportunities to advance their education, [wgu cybersecurity cost ](https://mineselect.com/cybersecurity/wgu-cybersecurity-admissions)is worth looking into as an affordable option for earning a degree in a high-demand field

**Comment by muqaddas608** [comment]
The[ Pisay GWA Calculator](https://gwacalculatores.com/) is a tool used by Philippine Science High School (Pisay) students to compute their General Weighted Average (GWA) based on subject grades and their corresponding weightings.
